************* Module multiprocessing.synchronize
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:29:0: C0325: Unnecessary parens after 'except' keyword (superfluous-parens)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:30:4: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:46:0: E0102: class already defined line 28 (function-redefined)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:46:0: R0205: Class 'SemLock' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:48:12: W0212: Access to a protected member _RandomNameSequence of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:57:16: C0103: Variable name "sl" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:72:16: W0212: Access to a protected member _after_fork of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:72:16: W0212: Access to a protected member _semlock of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:79:12: C0415: Import outside toplevel (resource_tracker.register) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:55:12: W0612: Unused variable 'i' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:86:8: C0415: Import outside toplevel (resource_tracker.unregister) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:102:8: C0103: Variable name "sl" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:104:12: C0103: Variable name "h" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:106:12: C0103: Variable name "h" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:116:26: W0212: Access to a protected member _config of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:129:15: W0212: Access to a protected member _get_value of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:134:15: W0703: Catching too general exception Exception (broad-except)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:145:8: W0233: __init__ method from a non direct base class 'SemLock' is called (non-parent-init-called)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:145:8: W0233: __init__ method from a non direct base class 'SemLock' is called (non-parent-init-called)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:144:4: W0231: __init__ method from base class 'Semaphore' is not called (super-init-not-called)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:150:15: W0703: Catching too general exception Exception (broad-except)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:176:15: W0703: Catching too general exception Exception (broad-except)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:202:15: W0703: Catching too general exception Exception (broad-except)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:210:0: R0205: Class 'Condition' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:243:15: W0703: Catching too general exception Exception (broad-except)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:248:15: W0212: Access to a protected member _is_mine of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:248:15: W0212: Access to a protected member _semlock of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:255:16: W0212: Access to a protected member _count of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:255:16: W0212: Access to a protected member _semlock of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:256:12: W0612: Unused variable 'i' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:270:4: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:271:15: W0212: Access to a protected member _is_mine of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:271:15: W0212: Access to a protected member _semlock of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:289:16: W0612: Unused variable 'i' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:321:0: R0205: Class 'Event' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:360:0: R0902: Too many instance attributes (8/7) (too-many-instance-attributes)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:362:4: W0231: __init__ method from base class 'Barrier' is not called (super-init-not-called)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:363:8: C0415: Import outside toplevel (struct) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:364:8: C0415: Import outside toplevel (heap.BufferWrapper) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/multiprocessing/synchronize.py:18:0: C0411: standard import "import time" should be placed before "import _multiprocessing" (wrong-import-order)

-----------------------------------
Your code has been rated at 8.14/10

